docs/example/README.rst

```diff
@@ -0,0 +1,4 @@
+Tutorial
+================
+
+Hello you
```

Comment:
Is this wanted?

Reply:
nothing special, a readme is need for sphinx-gallery